cmd/evm/testdata/10/txs.json

```diff
@@ -11,8 +11,8 @@
     "secretKey" : "0x41f6e321b31e72173f8ff2e292359e1862f24fba42fe6f97efaf641980eff298",
     "chainId" : "0x1",
     "type" : "0x2",
-    "feeCap" : "0xfa0",
-    "tip" : "0x0",
+    "gasFeeCap" : "0xfa0",
+    "gasTipCap" : "0x0",
```

Comment:
I guess these should really map the actual json names, rather than what we use internally

Reply:
When you run what's in the readme for that folder. But I just checked, it works fine now!